src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from 'react';
+
+import { Person } from '../../types';
+import { Loader } from '../../components/Loader';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../../components/PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+
+  const handleLoadPeople = async () => {
+    try {
+      const peopleFromServer = await getPeople();
+
+      setPeople(peopleFromServer);
+      setIsLoading(false);
+      setHasError(false);
+    } catch {
+      setHasError(true);
+      setIsLoading(false);
+    }
```

Comment:
Maybe it would be better)
```suggestion
    try {
      const peopleFromServer = await getPeople();

      setPeople(peopleFromServer);
      setHasError(false)
    } catch {
      setHasError(true);
    } finally {
      setIsLoading(false);
```